model/config.go

```diff
@@ -2201,6 +2203,8 @@ type PluginSettings struct {
 	ClientDirectory          *string `restricted:"true"`
 	Plugins                  map[string]map[string]interface{}
 	PluginStates             map[string]*PluginState
+	EnableMarketplace        *bool
+	MarketplaceURL           *string
```

Comment:
went with `URL` as opposed to `Url`, which unfortunately is inconsistent with 
https://github.com/mattermost/mattermost-server/pull/11933/files#diff-2279f86634da3f787f279961af76713aR2200 `AllowInsecureDownloadUrl` 🤦‍♂ , but is more aligned with go's convention. Breaks my 💔  to see the inconsistency.

Reply:
0/5, but should we keep the inconsistency with the rest of this file/struct?

Reply:
there is already a mix of `Url` `URL` within this file, however maybe it's worth keeping it the same within this struct. 👍